webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -161,18 +160,19 @@ def __init__(self, stored_data=None, render_info=None, *args, **kwargs):
             header_title=_('form.eligibility.header-title'),
             stored_data=stored_data,
             render_info=render_info,
+            form_storage=CookieStorage(),
             *args,
             **kwargs)
 
     def _main_handle(self):
         super()._main_handle()
         # Remove all eligibility data as the flow is restarting
         stored_data = {}
-        self._override_session_data(stored_data, session_data_identifier=self.session_data_identifier)
+        self._override_storage_data(stored_data, data_identifier=self.session_data_identifier)
         self.render_info.additional_info['next_button_label'] = _('form.eligibility.check-now-button')
 
-    def _override_session_data(self, stored_data, session_data_identifier=None):
-        override_data_in_cookie(stored_data, session_data_identifier)
+    def _override_storage_data(self, stored_data, data_identifier=None):
+        CookieStorage().override_data(stored_data, data_identifier)
```

Comment:
Why don't we use `self.form_storage.override_data` here instead of the_override_storage_data method?

Reply:
Or is this because of the implementation in the steuerlotse_step.py.

Reply:
done -> we need it in the steuerlotse_step because EligibilityStartDisplaySteuerlotseStep needs that functionality.